addons/website/static/src/js/content/menu.js

```diff
@@ -814,6 +815,39 @@ publicWidget.registry.MegaMenuDropdown = publicWidget.Widget.extend({
         this.options.wysiwyg?.odooEditor.observerActive("moveMegaMenu");
     },
 
+    /**
+     * @private
+     */
+    _updateActiveMenuLinks() {
+        // Prevent having several active links in the menu.
+        if (this.el.querySelector(".navbar:nth-child(1) a.nav-link.active")) {
+            return;
+        }
+        const currentPathName = window.location.pathname;
+
+        // Check and update the active state of menu items based on the current
+        // page
+        const megaMenuEls = this.el.querySelectorAll(".o_mega_menu");
+        let matchingLink = null;
+        megaMenuEls.forEach((dropdownMenusEl, position) => {
```

Comment:
```suggestion
        megaMenuEls.forEach((megaMenuEl, position) => {
```
it's more readable like that